src/components/ArticleList.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { IonContent } from '@ionic/react';
+
+const ArticleList = () => {
+  return (
+    <IonContent align="center">
+      This is where the articles go
+    </IonContent>
+  )
+}
+
```

Comment:
Is this just initial testing?

Reply:
yes, we were making sure components could render properly but this is just a scaffold

Reply:
cool